### PR TITLE
Adding final touches to site for optional check

### PR DESCRIPTION
**Pull request by RyanA760**
@bontgoy Hey just wanted to see if I am on the right track wanted to have you check it before merging.  Pretty sure I hit all the rubric and some extra to practice a few things.

### CODE REVIEW

File: guide.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	<fieldset>
+		<table border=2>
+			<tr>
+				<th>Brand</th>
+				<th>Model</th>
+				<th>Type</th>
+			</tr>
+			<tr>
+				<td>CLYW</td>
+				<td><a href="Images/summit.jpg">Summit</a></td>
```
Comment (Bontgoy) [inline guide.html]: these are anchor tags.  if you want to link an image to a page you should look into the `` tag

File: guide.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+				<td>Unresponsive</td>
+			</tr>
+			<tr>
+				<td>YoYoFactory</td>
+				<td><a href="Images/shutter.jpg">Shutter</a></td>
```
Comment (Bontgoy) [inline guide.html]: same

File: guide.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+				<td>Unresponsive</td>
+			</tr>
+			<tr>
+				<td>Duncan</td>
+				<td><a href="Images/Vintage">Imperial</a></td>
```
Comment (Bontgoy) [inline guide.html]: same

File: players.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        <option value="thrower">Yamata Murata</option>
+        <option value="thrower">Zach Gormley</option>
+        <option value="thrower">Gentry Stein</option>
+        <option value="thrower">Beatriz Gallegos Gracia</option>
+        <option value="thrower">Tessa Piccillo</option>
+        <option value="thrower">Stephanie Haight</option>
+        <option value="thrower">Chloe Monsonego</option>
+      </select>
+		</div>
+	</fieldset>
```
Comment (Bontgoy) [inline players.html:41]: Your form should have a submit button.  Also, the make sure form method is a `get` not a `post` at this time.

**Comment by RyanA760** [comment]
@bontgoy Hey Tyler I updated the things you pointed out, was a long day yesterday and finished later at night which is why I wanted to make sure it was good the html validator said it was OK.  Just wanted to get your view hopefully I didn't miss anything this time.

**Comment by Bontgoy** [comment]
Looks good @RyanA760 thanks for updating the form!

**Pull request merged**